tools/github-promote-release.ts

```diff
@@ -59,7 +61,10 @@ program
   .requiredOption('--platform <platform>')
   .action(async (options: Options) => {
     try {
-      await removePreRelease(options)
+      const promotionContext = await removePreRelease(options)
+      if (promotionContext) {
+        console.log(JSON.stringify(promotionContext))
```

Comment:
:thinking: Personally I would find it better to only notify about the release (including a link) that was promoted and not repost the whole release notes, as I think this makes the channel more cluttered and confusing why the same release notes are posted again. So I would imagine something like this:
```suggestion
        console.log(`[${release.name}](${release.html_url})`)
```
However, that is just my personal opinion. What do you think?

Reply:
Sounds good 👍